waves/src/theme.tsx

```diff
@@ -0,0 +1,61 @@
+import { extendTheme } from "@chakra-ui/react";
+
+const theme = extendTheme({
+    textStyles: {
+        actionable: {
+            fontSize: "lg",
+            color: "gray.500",
+        },
+        info: {
+            fontSize: "sm",
+            color: "gray.500",
+        },
+        assetSelect: {
+            fontSize: "md",
+            color: "gray.500",
+        },
+    },
+    swapButton: {
+        baseStyle: {
+            colorScheme: "teal",
+            size: "lg",
+            bg: "#304FFE",
+            rounded: "md",
+            width: "200px",
+        },
+    },
+    components: {
+        Button: {
+            baseStyle: {
+                bg: "#304FFE",
+                fontWeight: "bold",
+                fontColor: "green",
+                color: "green",
+                _hover: {
+                    bg: "blue.300",
+                },
+            },
+
+            sizes: {
+                lg: {
+                    h: "56px",
+                    fontSize: "lg",
+                    px: "32px",
+                },
+            },
+            // Custom variant
+            variants: {
+                "main_button": {
+                    h: "50px",
```

Comment:
Uhm, fixed values in `px` are quite the anti-pattern. Please either use `rem` or even better, let the button resize itself to how big it needs to be and then apply padding.

Reply:
I find formatting in css really hard. I rather leave it as is and if we want to change the style we get a someone who knows their art?

Reply:
> I find formatting in css really hard.

Then I would suggest to not add any customization. By default, the button should be big enough to fit its content.

Bad customization is worse than no customization IMO.

Reply:
I was able to remove some...